docs/source/algorithms/LoadDNSEvent-v1.rst

```diff
@@ -0,0 +1,78 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+
+This algorithm loads a DNS mesytec psd listmode file into an :ref:`EventWorkspace <EventWorkspace>`, it does not load the instrument definition.
+
+**Output**
+
+    :ref:`EventWorkspace <EventWorkspace>` with the X-axis in the Time-of-flight units.
+
+**ChopperChannel**
+is the input channel to wich the chopper was connected, must also be set in mesydaq gui.
```

Comment:
Could we rephrase this to:
`The chopper input channel, must also...`

Reply:
sure, changed it to: The chopper input channel as set in mesydaq gui.